src/ProjectService.Mappers/Models/ProjectInfoMapper.cs

```diff
@@ -25,11 +25,11 @@ public ProjectInfo Map(DbProject value, string departmentName)
                 ShortName = value.ShortName,
                 Description = value.Description,
                 ShortDescription = value.ShortDescription,
-                Department = new DepartmentInfo
+                Department = value.DepartmentId != Guid.Empty ? new DepartmentInfo
```

Comment:
why department id can bo guid empty?

Reply:
Why not, if there will be such response, we handle it correctly

Reply:
we cannot have a department with an empty guid, so if we receive an empty one, this is a service error, and hiding it is strange